assets/js/pages/article/read/ReadArticle.vue

```diff
@@ -45,17 +45,23 @@
     import ArticleContent from "pages/article/read/ArticleContent";
 
     import changePageTitleMixin from "mixins/changePageTitleMixin";
+    import ArticleTitle from "../header/ArticleTitle";
 
     export default {
         name: "ReadArticle",
-        components: {ArticleContent, ArticleNotFound, TableOfContent},
+        components: {ArticleTitle, ArticleContent, ArticleNotFound, TableOfContent},
         mixins: [changePageTitleMixin],
         data() {
             return {
                 notFound: false
             }
         },
         computed: {
+            currentArticleMetadata() {
+                return {
+                    title: this.$store.getters.getCurrentArticle?.file?.metadata?.title
```

Comment:
Why returning an object ? Maybe we should just return this.$store.getters.getCurrentArticle?.file?.metadata here

Reply:
The idea is to prepare multiple fields, that's why the currentArticleMetadata is not named currentArticleTitle :-)

What do you think about this?